buildpacks/nodejs-engine/tests/integration_test.rs

```diff
@@ -0,0 +1,59 @@
+#[cfg(test)]
+use libcnb_test::{BuildpackReference, IntegrationTest};
+use std::time::Duration;
+use ureq;
+
+#[test]
+fn test_node_with_indexjs() {
+    IntegrationTest::new(
+        "heroku/buildpacks:20",
+        "../../test/fixtures/node-with-indexjs",
+    )
+    .buildpacks(vec![BuildpackReference::Crate])
```

Comment:
This `.buildpacks()` call is equivalent to the default value, so isn't needed:
https://github.com/Malax/libcnb.rs/blob/5b9495d84f605018f9ff61c3b72ca422d975acde/libcnb-test/src/lib.rs#L118

```suggestion
```